config/src/config/mempool_config.rs

```diff
@@ -43,6 +44,7 @@ impl Default for MempoolConfig {
             system_transaction_timeout_secs: 600,
             system_transaction_gc_interval_ms: 60_000,
             shared_mempool_validator_broadcast: true,
+            broadcast_buckets: vec![0, 101, 201],
```

Comment:
we should check with ecosystem folks or @vgao1996 on what they expect gas price to average out, but this should probably be exponential. i.e like:
[0, 10, 100] or [0,30,1000]

or at least an order of magnitude gap:
[0, 100, 1000]

Reply:
Yes, we need a discussion on these values!

I believe min gas starts at 100, and so IMO we should have our first bucket just for that.

Reply:
yup the minimum is 100

Reply:
The minimum gas unit price is a configurable value so I feel like maybe we shouldn't hardcode it...